cvereporter/report.py

```diff
@@ -0,0 +1,50 @@
+from cyclonedx.exception import MissingOptionalDependencyException
+from cyclonedx.factory.license import LicenseFactory
+from cyclonedx.model import OrganizationalEntity, XsUri
+from cyclonedx.model.bom import Bom
+from cyclonedx.model.component import Component, ComponentType
+from cyclonedx.model.impact_analysis import ImpactAnalysisAffectedStatus
+from cyclonedx.model.vulnerability import Vulnerability, VulnerabilitySource,VulnerabilityScoreSource, VulnerabilityRating, VulnerabilitySeverity, BomTarget, BomTargetVersionRange
+from cyclonedx.output import make_outputter, LATEST_SUPPORTED_SCHEMA_VERSION
+from cyclonedx.output.json import JsonV1Dot4
+from cyclonedx.schema import SchemaVersion, OutputFormat
+from cyclonedx.validation.json import JsonStrictValidator
+from cyclonedx.validation import make_schemabased_validator
+from datetime import datetime
+
+# based on sample code from https://cyclonedx-python-library.readthedocs.io/en/latest/examples.html
+lc_factory = LicenseFactory()
+bom = Bom()
+bom.metadata.component = root_component = Component(
+    name='Eclipse Temurin',
+    type=ComponentType.APPLICATION,
```

Comment:
Our type is PLATFORM which, [from the docs (p.17)](https://cyclonedx.org/guides/sbom/OWASP_CycloneDX-SBOM-Guide-en.pdf), is:

> "A runtime environment which interprets or executes software. This may include runtimes such as those that execute bytecode or low-code/nocode application platforms. "

Reply:
I don't see it in the component type options in the library:

```
    APPLICATION = 'application'
    CONTAINER = 'container'
    DEVICE = 'device'
    FILE = 'file'
    FIRMWARE = 'firmware'
    FRAMEWORK = 'framework'
    LIBRARY = 'library'
    OPERATING_SYSTEM = 'operating-system'
```